doc/architecture/sensor_view_configuration.adoc

```diff
@@ -1,18 +1,33 @@
 = Sensor view configuration
 
+The sensor view is flexibly defined to support different kinds of sensor models with appropriate input.
+The sensor view configuration defines the configuration of a particular sensor view.
+
+This message is used in the initialization phase of a simulation to negotiate the sensor view configuration for a particular sensor view input.
```

Comment:
@pmai: What do you mean with "This message"? 
Is it the "sensor-view-configuration message"?